exareme2/controller/services/api/algorithm_spec_dtos.py

```diff
@@ -121,6 +122,18 @@ def _get_data_model_input_data_specification_dto():
     )
 
 
+def _get_valiadtion_datasets_input_data_specification_dto():
+    return InputDataSpecificationDTO(
+        label="Set of data to validate.",
+        desc="The set of data to validate the algorithm model on.",
+        types=[InputDataType.TEXT],
+        notblank=True,
+        multiple=True,
+        stattypes=None,
+        enumslen=None,
+    )
+
+
```

Comment:
forgotten line, how come the linter doesn't break?

Reply:
It somehow did not catch it